<launch>

	<arg name="hand" /> <!-- left, right, or both -->
	<param name="hand_mode" type="string" value="$(arg hand)" />

	<param name="hand_switch" type="int" value="1" /> <!-- used for "both" mode, 1 is right and -1 is left -->

	<param name="plane_origin" value="" />
	<param name="plane_x_vector" value="" />
	<param name="plane_y_vector" value="" />

	<param name="plane_eq_params" value="" />

	<node name="robot_interface_motion" pkg="group7_proj2" type="robot_interface_motion.py" output="screen" />

	<node name="controller_motion" pkg="group7_proj2" type="controller_motion.py" output="screen"/>

	<include file="$(find collision_checker)/launch/collision_server.launch" />

	<!--<include file="$(find collision_checker)/collision_server.launch"><arg name="visualize" value="true"/></include>-->

</launch>
